src/main/java/org/opentripplanner/updater/stoptime/TimetableSnapshotSource.java

```diff
@@ -478,6 +515,14 @@ private List<StopLocation> checkNewStopTimeUpdatesAndFindStops(
     Long previousTime = null;
     final List<StopTimeUpdate> stopTimeUpdates = tripUpdate.getStopTimeUpdateList();
     final List<StopLocation> stops = new ArrayList<>(stopTimeUpdates.size());
+    var tripId = Optional
```

Comment:
Should we add the feedId to this already? And should the else branch match the one above?

Reply:
I've converted a few calls to use the feed-scoped id.